crates/runtime/src/libcalls.rs

```diff
@@ -419,6 +426,14 @@ pub unsafe extern "C" fn wasmtime_externref_global_get(
             let raw = externref.as_raw();
             let (activations_table, module_info_lookup) =
                 (*instance.store()).externref_activations_table();
+
+            // We must maintain the invariant that all `externref`s on the stack
+            // have an entry in the activations table. This invariant is
+            // asserted when we GC. So we need to ensure that this `externref`
+            // is in the table *before* we GC, even though `insert_with_gc` will
+            // ensure that it is in the table *after* the GC.
+            activations_table.insert_without_gc(externref.clone());
```

Comment:
I don't think I understand why this is necessary, but was this required from fuzzing?

Unlike the `wasmtime_activations_table_insert_with_gc` function above the `externref` here is not on the stack in wasm, it's only on the stack here in Rust, and it's already rooted with the `Some(externref)` which should be an owned value.

I think that means that there's no need to manually insert it into the table because a stack walk otherwise won't find this (unless it would otherwise find it for some other reason)

Reply:
No, this isn't required, it was just me being overly cautious. I'll remove it and force push.